ecosystem/sep-0001.md

```diff
@@ -129,6 +129,7 @@ is_unlimited | boolean | The number of tokens is dilutable at the issuer's discr
 is_asset_anchored | boolean | `true` if token can be redeemed for underlying asset, otherwise `false`
 anchor_asset_type | string | Type of asset anchored. Can be `fiat`, `crypto`, `stock`, `bond`, `commodity`, `realestate`, or `other`.
 anchor_asset | string | If anchored token, code / symbol for asset that token is anchored to. E.g. USD, BTC, SBUX, Address of real-estate investment property.
+attestation_of_reserve | url | URL to a page hosting attestation(s) or other proof(s) of reserves, such as PDFs of third-party audits.
```

Comment:
Follow to https://github.com/stellar/stellar-protocol/pull/850#discussion_r565590305 I suggest changing this a little:
```suggestion
attestation_of_reserve | url | URL to attestation or other proof, evidence, or verification of reserves, such as third-party audits.
```